src/Styleguide/Components/FollowArtistPopover/FollowArtistPopoverRow.tsx

```diff
@@ -10,6 +10,7 @@ import { get } from "Utils/get"
 
 interface Props extends ContextProps {
   artist: FollowArtistPopoverRow_artist
+  suggestedIds: string[]
```

Comment:
This will be passed from the parent as a shared properties between all rows in the popover.